tests/clients/sts_test_client.go

```diff
@@ -28,15 +27,14 @@ func NewSTSTestClient(t *testing.T, url string) *STSTestClient {
 }
 
 func (s *STSTestClient) WaitForHealthy(timeout time.Duration) {
-	testutils.WaitFor(s.T, time.Second*10, timeout, func() bool {
-		logrus.Debugf("Waiting for sts service to be healthy")
+	log.Printf("[DEBUG] Waiting for sts service to be healthy")
+	for start := time.Now(); time.Since(start) < timeout; time.Sleep(time.Second * 10) {
 		if _, err := s.Client.GetHealth(); err != nil {
-			logrus.Debug(err)
-			return false
+			log.Printf(err.Error())
 		}
+	}
 
-		return true
-	})
+	log.Fatalf("[ERROR] Timeout reached after %v", timeout)
```

Comment:
This is a case where I can kinda see Dave Cheney's point re: not logging `[ERROR]`s, or at least labeling them as such. Reaching a timeout is an event that is handled by us (to some degree) and results in that test not passing, which is not necessarily unexpected behavior.

Reply:
Let's say I remove this line and the timeout condition happens. I'm not sure that the tests that call `sts.WaitForHealthy` would actually show failure or fail. It seems as though I should probably treat this as an informational level message like `log.Printf("[DEBUG] Timeout reached after %v", timeout)`

Reply:
yep that's what I was thinking

Reply:
:white_check_mark: